### PR TITLE
Removed :class:`~.GraphScene`, :class:`~.NumberLineOld` and parameters for :class:`~.ChangingDecimal`

### PR DESCRIPTION
**Pull request by hydrobeam**
## Motivation

Removing deprecated classes.

## Explanation for Changes

Removed `graph_scene.py` and `NumberLineOld` from `number_line.py`
Removed `GraphScene` specific tests in `text_axes_shift.py` and moved `test_graph_scene.py`'s test to `test_axes.py`.

## Further Comments

`GraphScene` is finally gone 🥳

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [x] My new functions/classes either have a docstring or are private
- [x] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [x] The PR is labeled correctly
- [x] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

**Review by jsonvillanueva — COMMENTED** [review]
Seems like that's all of the `GraphScene` and `NumberLine` deprecations, but there's also `numbers.py` which has a deprecation warning for the Animation `ChangingDecimal`. Could we also remove that in this PR to address all outstanding deprecations?

**Comment by hydrobeam** [comment]
It seems like `ChangingDecimal` has a deprecated paramater that doesn't even exist, so I've removed the warning for that class.

File: manim/animation/numbers.py [hunk 1]
```diff
@@ -1,23 +1,16 @@
 """Animations for changing numbers."""
 
-__all__ = ["ChangingDecimal", "ChangeDecimalToValue"]
+__all__ = ["ChangeDecimalToValue"]
```
Comment (jsonvillanueva) [inline manim/animation/numbers.py:3]:
```suggestion
__all__ = ["ChangingDecimal", "ChangeDecimalToValue"]
```

**Review by jsonvillanueva — APPROVED** [review]
LGTM

**Pull request merged**